README.md

```diff
@@ -45,9 +46,19 @@ const config = {
   experienceVersion: 'PRODUCTION',
 }
 
+const searcher = provideHeadless(config);
+
+searcher.setSessionTrackingEnabled(true);
+let sessionId = window.sessionStorage.getItem('sessionId');
```

Comment:
I guess we are just providing an example of client side rendering so no need to guard window or anything?